### ISSUE TITLE
Installing a new chart fails immediately....

### ISSUE THREAD

**Issue by natiki** [kubeapps/kubeapps#491]
Hi,

- Alpha-5, Minikube on Windows 10
- So following on from my #472 I decided to see if I could install a chart
- So I went to install:

![image](https://user-images.githubusercontent.com/4738140/44146563-828208d0-a0d3-11e8-83ee-ba9353681e6e.png)

changing only the http port to 8081 and got the immediate error as seen per the screenshot. No pods were even attempted it seems:
```
C:\Kubernetes>kubectl get po --all-namespaces -l created-by=kubeapps
No resources found.
```

**Comment by andresmgot** [comment]
Hi @natiki, sorry that command currently doesn't apply when Kubeapps is installed through the chart (I opened #494 to track that issue).

Does it work for you if you don't change the port? If that is the case probably Tiller is not able to deploy the chart. You can check the logs of Tiller executing `kubectl logs -n kube-system -l app=helm`. Note that we plan to show Tiller errors in the web application once #466 is merged.

**Comment by prydonius** [comment]
I believe this is the same issue as https://github.com/kubeapps/kubeapps/issues/472#issuecomment-413270378

**Comment by prydonius** [comment]
closing in favour of #472, reopen if this is a different issue!

**Comment by natiki** [comment]
this is ultimately being dealt with in #494. #472 is a permissions issue.

**Comment by prydonius** [comment]
thanks @natiki for clarifying that
